src/commonMain/kotlin/app/revanced/library/installation/command/AdbShellCommandRunner.kt

```diff
@@ -56,4 +60,6 @@ class AdbShellCommandRunner : ShellCommandRunner {
      * @param targetFilePath The target file path.
      */
     override fun move(file: File, targetFilePath: String) = device.push(file, RemoteFile(targetFilePath))
+
+    internal class ShellCmdFailure internal constructor() : Exception("Shell command execution failure")
```

Comment:
Naming of the class isnt consistent. We don't shorten names and append super type names to the class name:

```suggestion
    internal class ShellCommandRunnerException internal constructor() : Exception("Shell command execution failed")
```

Reply:
True. Moved to `ShellCommandRunner`